### PR TITLE
Remove unused local integration testing

### PR DESCRIPTION
**Pull request by bocekm**
We don't run integration tests locally anymore - we run them just in the Testing Farm nowadays.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -163,31 +163,8 @@ Also, you can check out our
...
-define which which tests will be executed. Tests are then executed using the
-[tmt](https://tmt.readthedocs.io/en/stable/index.html) tool. Tests themself are
-written in mostly in `Python`. However one can write the integration test in
-any other language, eg. in `Bash`.
-
-Finally, you can find out all the integration testing for Convert2RHEL at the
-[convert2rhel/tests](https://github.com/oamg/convert2rhel/tree/main/tests)
-folder.
+To learm more about our integration tests, see
+[tests/integration/README.md](https://github.com/oamg/convert2rhel/blob/main/tests/integration/README.md).
```
Comment (r0x0d) [inline CONTRIBUTING.md:167]: I don't think we need to remove most of the text here, just removing the instance of the word `locally` seems to be enough.

**Review by r0x0d — COMMENTED** [review]
Just one minor comment

File: CONTRIBUTING.md (hunk 1 above)
Comment (bocekm) [reply CONTRIBUTING.md:167]: I moved most of this text to `tests/integration/README.md`.

Before this PR details about integration tests were partially duplicate in this CONTRIBUTING.md file and the `tests/integration/README.md`.

This change makes it consistent with what we have in the CONTRIBUTING.md about unit tests - basically just linking to `unit_tests/README.md` for more information.

Comment (r0x0d) [reply CONTRIBUTING.md:167]: Got it. Makes sense to me now.

**Review by r0x0d — DISMISSED** [review]

**Review by kokesak — COMMENTED** [review]
Found just one typoo, otherwise it's OK.

**Review by kokesak — APPROVED** [review]

**Review by danmyway — APPROVED** [review]

**Comment by bocekm** [comment]
I'm merging this even before the the integration tests on OL8 haven't finished running. There's nothing in this PR that could affect the integration tests or the way convert2rhel functions.
And one of the integration test on epel-7 failed with `Timed out waiting for last boot time check (timeout=600)`. That should be fixed by #432 merged today.

**Pull request merged**